controllers/update_pods.go

```diff
@@ -131,7 +131,7 @@ func (updatePods) reconcile(ctx context.Context, r *FoundationDBClusterReconcile
 			return &requeue{message: "Requeueing reconciliation to replace pods"}
 		}
 
-		if !pointer.BoolDeref(cluster.Spec.AutomationOptions.DeletePods, true) {
+		if !pointer.BoolDeref(cluster.Spec.AutomationOptions.DeletePods, true) || r.PodLifecycleManager.GetDeletionMode(cluster) == fdbtypes.PodUpdateModeNone {
```

Comment:
If the field in AutomationOptions is deprecated, do we need to read it here, or can we take care of it in NormalizeClusterSpec?

Reply:
I think we can take care of that in the `NormalizeClusterSpec`.